src/components/ArticleSortOptions.jsx

```diff
@@ -22,11 +51,17 @@ function ArticleSortOptions() {
           <Select
             labelId="sort-by-label"
             id="sort-by"
-            value={""}
+            value={sortBy}
             label="Sort-By"
             onChange={handleSelect}
           >
-            <MenuItem value={""}>TBC</MenuItem>
+            {sortOptions.map((option) => {
+              return (
+                <MenuItem value={option} key={option}>
+                  {option}
+                </MenuItem>
+              );
+            })}
```

Comment:
If at all possible, I would recommend maybe having the 'asc' and desc' labels on the toggle rather than just order as it's unclear before you play around with it which way you need to toggle and what that will do to the order of the articles - just a UX suggestion :)